bounties_api/notifications/notification_client.py

```diff
@@ -379,9 +379,13 @@ def comment_received(self, bounty_id, event_date, uid, **kwargs):
         string_data = notification_templates['BountyCommentReceived'].format(
             bounty_title=bounty.title)
 
+        commenters = list(map(lambda c: c.user, bounty.comments.all()))
```

Comment:
In the future I'd like to solve this using `itertools.chain` and `operator.attrgetter` - for now we'll just keep it simple and add another map.